lib/modules/datasource/aws-rds/readme.md

```diff
@@ -77,3 +81,30 @@ spec:
   # amiFilter=[{"Name":"engine","Values":["mysql"]},{"Name":"engine-version","Values":["5.7"]}]
   engineVersion: 5.7.34
 ```
+
+#### Using Terraform, `aws-rds` datasource and Aurora MySQL
+
+Here is the Renovate configuration to use Terraform, `aws-rds` and Aurora MySQL:
+
+```json
+{
+  "customManagers": [
+    {
+      "description": "Update RDS",
+      "customType": "regex",
+      "fileMatch": [".+\\.tf$"],
+      "matchStrings": [
+        "\\s*#\\s*renovate:\\s*amiFilter=(?<lookupName>.+?) depName=(?<depName>.*) versioning=(?<versioning>.*)\\s*.*_version\\s*=\\s*\"(?<currentValue>.*)\""
+      ],
+      "datasourceTemplate": "aws-rds"
+    }
+  ]
+}
+```
```

Comment:
This is basically copy/pasted from:

- https://github.com/renovatebot/renovate/discussions/19928#discussioncomment-9868352

But I changed these bits:

- Ran the config through Prettier, and made it valid JSON
- Change the users's "old" `regexManagers` to the new term: `customManagers`

I'll need a review from someone who understand all this, to make sure it's correct. 😉